setup.cfg

```
@@ -24,14 +24,14 @@ install_requires =
     nibabel >= 3.0.1
     nipype >=1.3.1
     nitime
-    niworkflows ~= 1.1.6
+    niworkflows @ git+https://github.com/poldracklab/niworkflows.git@edeea81352f3f2b276f11e42e7829e1c5a03770f
     numpy
     pandas
     psutil >=5.4
     pybids ~= 0.9.4
     pyyaml
     sdcflows ~=1.1.0
-    smriprep ~= 0.5.1
+    smriprep @ git+https://github.com/bpinsard/smriprep.git@ef552bbad2418b558a2f639be502bcf98ec1cf3b
```

```suggestion
    smriprep ~= 0.5.2
```

---

Pin to poldracklab master to ensure we don't break things. Re-pin on release prep.

```suggestion
    smriprep @ git+https://github.com/poldracklab/smriprep.git@07a7ba3fce8a9953923bcd0fa6ffbffac401208d
```